New: Container Block.

RajkiranBagal opened the pull request:
### Description
 - Flex Based Container Block.
 - https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/5f7d294a-bc49-415c-9d76-99dd890e0910?view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Block Screencast
 https://share.bsf.io/bLuxqq77

### Types of changes
 New feature (non-breaking change which adds functionality)

### How has this been tested?
 - Test all the Settings.
 - Test by adding multiple container children blocks.

### Checklist:
- [x] My code is tested
- [x] I've included any necessary tests <!-- if applicable -->
- [x] I've added proper labels to this pull request <!-- if applicable -->

akshaylakh commented:
- General > Container > Width does not work in Editor responsive mode - https://share.bsf.io/eDulgYdQ
- General > Responsive Slider > when user changes units from PX to % min max value should change -  https://share.bsf.io/RBuyE7gx  [Ref PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/1191) 
- General > Container > Custom width & Min Height does not work in Editor responsive mode - 
- As we do have two width options default & custom. Can we have a toggle button to show at a time one only? - https://share.bsf.io/L1uWez4y
- Tooltips required for Direction, Justify Content, Align Item & Wrap - https://share.bsf.io/9ZuDRYdR

RajkiranBagal commented:
@akshaylakh 
General > Container > Width does not work in Editor responsive mode - https://share.bsf.io/eDulgYdQ
    - Fixed
General > Responsive Slider > when user changes units from PX to % min max value should change - https://share.bsf.io/RBuyE7gx Ref PR
    - Fixed
General > Container > Custom width & Min Height does not work in Editor responsive mode 
    - Fixed
As we do have two width options default & custom. Can we have a toggle button to show at a time one only? - https://share.bsf.io/L1uWez4y
    - It will look glitchy.
Tooltips required for Direction, Justify Content, Align Item & Wrap - https://share.bsf.io/9ZuDRYdR
    - We are going to add Readmore Panels in all blocks.
Style > Background > Gradient > Radial > Locations are missing - center center, center left - https://share.bsf.io/JruGnXYy
    - We are using core Gutenberg Gradient Picker, even our columns block do not have these options.
default values should be selected - https://share.bsf.io/NQux7e6g
    - Done
[25-Jan-2022 06:00:01 UTC] PHP Notice: Undefined index: boxShadowPositionCSS in C:\Users\aksha\Local Sites\test2\app\public\wp-content\plugins\ultimate-addons-for-gutenberg\includes\blocks\container\frontend.css.php on line 47
    - Fixed
